lms/templates/ux/reference/teams-base.html

```diff
@@ -17,8 +17,7 @@
     <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
       <a href="/" title="Home page" itemprop="url">
```

Comment:
@sarina I'm going to get rid of that attribute all together as it shouldn't even be there.